ios/Bridgeme.xcodeproj/xcshareddata/xcschemes/Yoda.xcscheme

```diff
@@ -66,7 +66,7 @@
       </AdditionalOptions>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Release"
+      buildConfiguration = "Debug"
```

Comment:
This file is about project.
If this file is removed, our project will not work.